src/zSDKClient.ts

```diff
@@ -0,0 +1,74 @@
+import shortid from 'shortid';
+
+import { Config, CreateZDAOParams, SDKInstance, zDAO, zNA } from './types';
+import { t } from './utilities/messages';
+import zDAOClient from './zDAOClient';
+import zNAClient from './zNAClient';
+
+class zSDKClient implements SDKInstance {
```

Comment:
Rename to `SDKInstanceClient` or if you want to rename `SDKInstance` to `Instance` you could just call it `InstanceClient` since we could infer that an "instance" would be of the SDK (root level subject of this project is an SDK)